contrib/babelfishpg_tds/src/backend/tds/tdssecure.c

```diff
@@ -245,6 +274,28 @@ SslHandShakeWrite(BIO * h, const char *buf, int size)
 	return (res - TDS_PACKET_HEADER_SIZE);
 }
 
+
+ssize_t
+test_ssl_handshake_write(BIO * h, char *buf, int size, TdsSecureSocketApiConst mock_socket_write)
+{
+
+    /*
+     *  Accessing SslHandShakeWrite() function directly from our testing functions created in the test_ssl_write.c file is not possible due to its static nature
+     *  This intermediary function serves as a gateway, allowing us to invoke the SslHandShakeWrite() function from our testing function
+     */
+
+    int res;
+    tds_secure_raw_write = mock_socket_write;
```

Comment:
should we reset it after the unit testing?

Reply:
Yes, have updated in the next revision.